src/Open3D/Visualization/Rendering/Filament/FilamentRenderToBuffer.h

```diff
@@ -0,0 +1,82 @@
+// ----------------------------------------------------------------------------
+// -                        Open3D: www.open3d.org                            -
+// ----------------------------------------------------------------------------
+// The MIT License (MIT)
+//
+// Copyright (c) 2020 www.open3d.org
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+// FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+// IN THE SOFTWARE.
+// ----------------------------------------------------------------------------
+
+#pragma once
+
+#include "Open3D/Visualization/Rendering/RenderToBuffer.h"
+
+namespace filament {
+class Engine;
+class Renderer;
+class Scene;
+class SwapChain;
+}  // namespace filament
+
+namespace open3d {
+namespace visualization {
+
+class FilamentRenderer;
+class FilamentScene;
+class FilamentView;
+
+class FilamentRenderToBuffer : public RenderToBuffer {
+public:
+    // Never create it alone, if you are using GUI app or another
+    // FilamentRenderer instance. Use Renderer::CreateBufferRenderer instead
```

Comment:
I think it means "use Renderer::CreateBufferRenderer instead of constructing manually unless you are not using the GUI app or FilamentRenderer". English was not the original author's native language...  Changed to: "Use Renderer::CreateBufferRenderer to instantiate  FilamentRenderToBuffer, unless you are NOT using open3d::gui or another FilamentRenderer instance."